tests/by-util/test_pwd.rs

```diff
@@ -26,7 +26,7 @@ fn test_deleted_dir() {
     let output = Command::new("sh")
         .arg("-c")
         .arg(format!(
-            "cd '{}'; mkdir foo; cd foo; rmdir ../foo; exec {} {}",
+            "cd '{}'; mkdir foo; cd foo; rmdir ../foo; exec '{}' {}",
```

Comment:
Is this change related to the changes in `mkdir`?

Reply:
The non-escaped pathnames caused problems in my machine when testing.